assets/sass/style.scss

```diff
@@ -1,7 +1,106 @@
 @import 'reset';
+html {
+  height: 100%;
+}
 body {
   margin: 0;
+  height: 100%;
+  width: 100%;
 }
 .main {
   width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+header {
+  height: 75px;
+  position: fixed;
+  top: 0;
+  background-image: url("../images/header-right.png");
+  background-position: right;
+  background-repeat: no-repeat;
+  background-size: 268px;
+  background-color: #2B3C43;
+  width: 100%;
+  span {
+    color: #D1D3D5;
+    position: relative;
+    top: 30%;
+    left: 45%;
+  }
+}
+footer {
+  position: fixed;
+  bottom: 0;
+  background-color: #2B3C43;
+  width: 100%;
+  height: 50px;
+  .footer-left, .footer-right {
+    background-size: contain;
+    display: inline-block;
+    height: 50px;
+    background-repeat: no-repeat;
+  }
+  .footer-left {
+    background-image: url("../images/footer-left.png");
+    background-position: left;
+    width: 20%;
+  }
+  .footer-right {
+    background-image: url("../images/footer-right.png");
+    background-position: right;
+    float: right;
+    width: 33%;
+  }
+  .footer_mid {
```

Comment:
Use a common naming convention for classes/ids. Avoid underscore and use hyphen for consistency.